numpy/core/numeric.py

```diff
@@ -2145,8 +2145,12 @@ def fromfunction(function, shape, **kwargs):
         The function is called with N parameters, where N is the rank of
         `shape`.  Each parameter represents the coordinates of the array
         varying along a specific axis.  For example, if `shape`
-        were ``(2, 2)``, then the parameters in turn be (0, 0), (0, 1),
-        (1, 0), (1, 1).
+        were ``(2, 2)``, then the parameters would in turn be
+            array([[ 0, 0 ],
+                   [ 1, 1 ]])
+        and
+            array([[ 0, 1 ],
+                   [ 0, 1 ]])
```

Comment:
Don't close it. If you add new commits, just push them to the branch you made this PR from and they'll show up here automatically.